### ISSUE TITLE
Do we need Git package as dependency?

### ISSUE THREAD

**Issue by igortg** [conda-forge/git-filter-repo-feedstock#5]
### Comment:

I'd say that in most environments, people get Git on a "provisioning" phase using more standard OS package managers (yum, apt, choco), instead of relying on conda-forge. So having this package with 66Kb depending on Git package (146Mb) seems not a good default. It can also lead to pretty messed up environments (two Gits on PATH) if people install this package without having a good look at its dependencies.

I'd suggest removing Git dependency since it's not usual to get Git from conda-forge.

**Comment by jdblischak** [comment]
I respectfully disagree. Conda is not simply an alternative to pip. Git is required for git-filter-repo to function, so it makes sense to list this as a required dependency.

For more context on the goals of conda and how it relates to other packaging options, I recommend this recent blog post series from last year:

* [Conda ≠ PyPI: Why Conda Is More Than a Package Manager](https://conda.org/blog/conda-is-not-pypi/)
* [Conda in the Packaging Spectrum: From pip to Docker to Nix](https://conda.org/blog/conda-pip-docker-nix/)
* [Practical Power: Reproducibility, Automation, and Layering with Conda](https://conda.org/blog/conda-practical-power/)

And if you really want to use an OS-installed Git within your conda environment, you can always install git-filter-repo with `pip`, which should install easily since it is a pure Python package.
